packages/expo-cli/src/commands/utils/deprecatedExtensionWarnings.ts

```diff
@@ -0,0 +1,100 @@
+import chalk from 'chalk';
+import program from 'commander';
+import findWorkspaceRoot from 'find-yarn-workspace-root';
+import glob from 'glob';
+import ora from 'ora';
+
+import { SilentError } from '../../CommandError';
+import Log from '../../log';
+import { confirmAsync } from '../../prompts';
+
+function queryExpoExtensionFilesAsync(projectRoot: string, ignore: string[]): Promise<string[]> {
+  return new Promise((resolve, reject) => {
+    glob(
+      '**/*.expo.@(js|jsx|ts|tsx)',
+      {
+        absolute: true,
+        ignore,
+        cwd: projectRoot,
+      },
+      (error, matches) => {
+        if (error) reject(error);
+        else resolve(matches);
+      }
+    );
+  });
+}
+
+export async function assertProjectHasExpoExtensionFilesAsync(
+  projectRoot: string,
+  checkNodeModules: boolean = false
+) {
+  if (checkNodeModules) {
+    return await assertModulesHasExpoExtensionFilesAsync(projectRoot);
+  }
+
+  //   console.time('count');
+  const matches = await queryExpoExtensionFilesAsync(projectRoot, [
+    `**/@(Carthage|Pods|node_modules|ts-declarations|.expo)/**`,
+    '@(ios|android|web)/**',
+  ]).catch(() => [] as string[]);
+  //   console.timeEnd('count');
+  if (!matches.length) {
+    return;
+  }
+
+  await promptMatchesAsync(matches);
+}
+
+async function assertModulesHasExpoExtensionFilesAsync(projectRoot: string) {
+  const spinner = ora('Checking project for deprecated features, this may take a moment.').start();
```

Comment:
```suggestion
  const spinner = ora('Checking project for deprecated features, this may take up to a minute depending on your project size and system.').start();
```

maybe something like this to give more context on the duration?

Reply:
long messages don't do well with spinners.